src/components/BookItem.js

```diff
@@ -8,19 +8,42 @@ const BookItem = ({ itemProp }) => {
   //   item_id, title, author, category,
   // } = itemProp;
   return (
-    <li>
-      <h1>{itemProp.category}</h1>
-      <h2>{itemProp.title}</h2>
-      <span>{itemProp.author}</span>
-      <button
-        onClick={() => {
-          dispatch(deleteBooks(itemProp.item_id));
-        }}
-        type="button"
-      >
-        Delete
-      </button>
-    </li>
+    <div className="book-item">
+      <div className="book">
+        <p className="category">{itemProp.category}</p>
+        <h2 className="title">{itemProp.title}</h2>
+        <p className="author">{itemProp.author}</p>
+        <div className="buttons">
+          <button type="button">Comments</button>
+          <button
+            onClick={() => {
+              dispatch(deleteBooks(itemProp.item_id));
+            }}
+            type="button"
+          >
+            Remove
+          </button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="progress">
+        <div className="circle" />
+        <div>
+          <p className="score">100%</p>
+          <p className="progress-status">Completed</p>
+        </div>
+
+      </div>
+      <div className="chapter">
+        <div className="chapter-text">
+          <h4>Current chapter</h4>
+          <p>Chapter 17</p>
+        </div>
+        <button className="progress-btn" type="button">UPDATE PROGRESS</button>
```

Comment:
- [ ] Give some padding to your `update progress` button so as to match the given design

Your design
![image](https://github.com/derrick1451/bookstore/assets/81932478/ec342588-8952-4153-9c40-77c281705058)

Given design
![image](https://github.com/derrick1451/bookstore/assets/81932478/94174cac-3f0f-4295-97b6-19ff5845f6c7)

## Note that the design will improve when the right fonts are used